tools/cockpit.spec

```diff
@@ -647,6 +648,7 @@ Requires: %{name}-bridge >= %{required_base}
 Requires: %{name}-shell >= %{required_base}
 Requires: /usr/bin/docker
 Requires: python2
+BuildArch: noarch
```

Comment:
This is inside an `%ifarch`, which should then probably be dropped for a noarch package. But more importantly, docker does not exist on all architectures,  so e. g. on powerpc there would now be a cockpit-docker package with an unsatisfiable docker dependency. Do we need this for the ostree tests, or just ostree itself? If the latter, I'd rather leave this part out.

Reply:
Ah, good point.  I was just inspecting the package contents, we don't need this to be noarch for anything.